Makefile

```diff
@@ -27,12 +27,10 @@ BOOTSTRAP_MANIFESTS =	sapi_manifests/registrar/template
 
 NODEUNIT_TESTS =	$(notdir $(wildcard test/*.test.js))
 
-NODE_PREBUILT_VERSION =	v0.10.48
-# sdc-*-multiarch 15.4.1.
-NODE_PREBUILT_IMAGE	= 18b094b0-eb01-11e5-80c1-175dac7ddf02
-ifeq ($(shell uname -s),SunOS)
-	NODE_PREBUILT_TAG =	zone64
-endif
+NODE_PREBUILT_TAG       = zone
```

Comment:
Why s/zone64/zone? It should stay at zone64. While it is hard to track, electric-moray was moved to a 64-bit node explicitly in https://jira.joyent.us/browse/MORAY-184  ... and unfortunately that ticket didn't give good reasoning (it gives *no* reasoning).  My recollection is to allow it to have a higher process mem cap for potentially huge in-memory hash ring for larger deployments.

Reply:
no purpose *at all* - this was copied from buckets-api